src/Sylius/Bundle/AdminBundle/Resources/views/Crud/Index/_header.html.twig

```diff
@@ -8,7 +8,9 @@
         <div class="ui right floated buttons">
             {% if definition.actionGroups.main is defined and definition.getActions('main')|length > 0 %}
                 {% for action in definition.getActions('main') %}
-                    {{ sylius_grid_render_action(resources, action, null) }}
+                    {% if action.enabled %}
```

Comment:
You can put the `for` and `if` on single line http://twig.sensiolabs.org/doc/2.x/tags/for.html#adding-a-condition